README.md

```diff
@@ -1,12 +1,105 @@
-# reactor-netty
+# Reactor Netty
 
 [![Join the chat at https://gitter.im/reactor/reactor](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/reactor/reactor?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![Reactor Netty](https://maven-badges.herokuapp.com/maven-central/io.projectreactor.netty/reactor-netty/badge.svg?style=plastic)](http://mvnrepository.com/artifact/io.projectreactor.netty/reactor-netty) [ ![Download](https://api.bintray.com/packages/spring/jars/io.projectreactor.netty/images/download.svg) ](https://bintray.com/spring/jars/io.projectreactor.netty/_latestVersion)
+[![Reactor Netty](https://maven-badges.herokuapp.com/maven-central/io.projectreactor.netty/reactor-netty/badge.svg?style=plastic)](https://mvnrepository.com/artifact/io.projectreactor.netty/reactor-netty) [ ![Download](https://api.bintray.com/packages/spring/jars/io.projectreactor.netty/images/download.svg) ](https://bintray.com/spring/jars/io.projectreactor.netty/_latestVersion)
```

Comment:
you could also use shields.io, they have Maven Central integration:
https://shields.io/#/examples/version

Reply:
done